subprojects/docs/src/docs/release/notes.md

```diff
@@ -32,6 +32,23 @@ details of 2
 
 ## n
 -->
+## Security 
+
+### Removed debug logging for environment variables
+
+Gradle allows you to start processes in various ways, for example via `Project.exec`, `Project.javaexec`, running tests or starting a Gradle daemon.
+When you enabled debug logging for the Gradle process, then it used to log all environment variables of the process about to start.
```

Comment:
I propose to separate the paragraph explaining the problem in the previous version as follows:

"Previously, when debug level was enabled, Gradle used to log all environment variables when starting a process such as a test, Gradle daemon, or `Project.exec`. In practice, this means most of the builds logged environment variables on debug level. "

Reply:
I used a slight variation of your suggestion.